db/migrate/20220330123815_move_script_based_actions_to_class.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
```

Comment:
While this controls a data change, it is data-agnostic and matches changes in the code. Will automatically convert to the new step handlers.